typings/index.d.ts

```diff
@@ -1701,6 +1703,11 @@ declare module 'discord.js' {
 		splash?: Base64Resolvable;
 	};
 
+	type GuildEmbedData = {
+		enabled: boolean;
+		channel?: GuildChannel;
```

Comment:
This should be a `GuildChannelResolvable` 👀

Reply:
Nice catch! Resolved in 4c8226a30895bc9b96fc20f6ae93067db7cf57e1